libs/dotnet-sdk/README.md

```diff
@@ -0,0 +1,129 @@
+## DotNet SDK
+
+## Installation
+
+```powershell
+dotnet add package Featureboard.DotnetSdk
+```
+
+
+## Setup
+
+1) Create a Features model
+```csharp
+using Featureboard.DotnetSdk.Models;
+
+public class WeatherFeatures : IFeatures
+{
+  public bool WeatherImperial { get; set; }
+}
+```
+
+2) Implement the audance provider
```

Comment:
Can we add more info here to say it can pull from the user's context etc, not just query string. In fact query string is bad practice because it allows the user to change anything (unless that's intended, as this example is)

Reply:
Updated example to pull from claims